content/docs/18-knowledgebase/00-how-to/01-how-to-setup-kube-api-audit-logging.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "How to Configure kube-apiserver audit logging"
+metaTitle: "Configuring the kube-apiserver audit logging featureo"
+metaDescription: "KnowledgeBase with How to"
```

Comment:
Let's add a description that helps search engines identify the content and improve our SEO score.